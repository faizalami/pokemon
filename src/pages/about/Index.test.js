import { render } from '@testing-library/react';
import AboutIndex from './Index';

test('About page matches snapshot', () => {
  expect(render(<AboutIndex/>)).toMatchSnapshot();
});
