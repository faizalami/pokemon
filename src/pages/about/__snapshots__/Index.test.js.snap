// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About page matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": .emotion-0 {
  list-style-type: disc;
}

<body>
    <div>
      <article>
        <p>
          An entertaining Pokemon catalogue app by Faizal Amiruddin.
        </p>
        <h2>
          Thanks to
        </h2>
        <ul
          class="emotion-0"
        >
          <li>
            <a
              href="https://www.flaticon.com/free-icons/pokemon"
              title="pokemon icons"
            >
              Pokemon icons created by Those Icons - Flaticon
            </a>
          </li>
          <li>
            <a
              href="https://www.flaticon.com/free-icons/pokemon"
              title="pokemon icons"
            >
              Pokemon icons created by Nikita Golubev - Flaticon
            </a>
          </li>
          <li>
            <a
              href="https://pokeapi.co/docs/graphql"
              title="PokéAPI"
            >
              PokéAPI - GraphQL v1beta
            </a>
          </li>
          <li>
            <a
              href="https://msikma.github.io/pokesprite/overview/misc.html"
              title="PokéSprite"
            >
              PokéSprite
            </a>
          </li>
        </ul>
      </article>
    </div>
  </body>,
  "container": .emotion-0 {
  list-style-type: disc;
}

<div>
    <article>
      <p>
        An entertaining Pokemon catalogue app by Faizal Amiruddin.
      </p>
      <h2>
        Thanks to
      </h2>
      <ul
        class="emotion-0"
      >
        <li>
          <a
            href="https://www.flaticon.com/free-icons/pokemon"
            title="pokemon icons"
          >
            Pokemon icons created by Those Icons - Flaticon
          </a>
        </li>
        <li>
          <a
            href="https://www.flaticon.com/free-icons/pokemon"
            title="pokemon icons"
          >
            Pokemon icons created by Nikita Golubev - Flaticon
          </a>
        </li>
        <li>
          <a
            href="https://pokeapi.co/docs/graphql"
            title="PokéAPI"
          >
            PokéAPI - GraphQL v1beta
          </a>
        </li>
        <li>
          <a
            href="https://msikma.github.io/pokesprite/overview/misc.html"
            title="PokéSprite"
          >
            PokéSprite
          </a>
        </li>
      </ul>
    </article>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
